scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

```diff
@@ -0,0 +1,225 @@
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-create-performance-profiles[Creating a performance profile]
+
+* xref:../../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-du-configuring-host-firmware-requirements_sno-configure-for-vdu[Configuring host firmware for low latency and high performance]
+
+* xref:../../installing/install_config/enabling-cgroup-v1.adoc#nodes-clusters-cgroups-2-install_nodes-cluster-cgroups-1[Enabling Linux cgroup v1 during installation]
+
+include::modules/telco-core-service-mesh.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../service_mesh/v2x/ossm-about.adoc#ossm-about[About OpenShift Service Mesh]
+
+include::modules/telco-core-networking.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/understanding-networking.adoc#understanding-networking[Understanding networking]
+
+include::modules/telco-core-cluster-network-operator.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/networking_operators/cluster-network-operator.adoc#nw-cluster-network-operator_cluster-network-operator[Cluster Network Operator]
+
+include::modules/telco-core-load-balancer.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/networking_operators/metallb-operator/about-metallb.adoc#nw-metallb-when-metallb_about-metallb-and-metallb-operator[When to use MetalLB]
+
+include::modules/telco-core-sr-iov.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/hardware_networks/about-sriov.adoc#about-sriov[About Single Root I/O Virtualization (SR-IOV) hardware networks]
+
+* xref:../../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices]
+
+* xref:../../networking/hardware_networks/configuring-sriov-device.html#nw-sriov-nic-mlx-secure-boot_configuring-sriov-device[Configuring the SR-IOV Network Operator on Mellanox cards when Secure Boot is enabled]
+
+include::modules/telco-core-nmstate-operator.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/networking_operators/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator]
+
+include::modules/telco-core-logging.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../observability/logging/logging-6.0/log6x-about.adoc#log6x-about[About logging]
+
+include::modules/telco-core-power-management.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../rest_api/node_apis/performanceprofile-performance-openshift-io-v2.adoc#spec-workloadhints[Performance Profile]
```

Comment:
Is the right link text? The link goes to a specific module in the PerformanceProfiule schema doc.